snooty/parser.py

```diff
@@ -1002,6 +1007,23 @@ def validate_tabs_children(self, node: n.Directive) -> None:
         node.children = new_children
         return
 
+    def validate_icon_role(self, node: docutils.nodes.Node) -> None:
+        """
+        Validate target for icon role
+        Checks for included icon file in root path
+        """
+        if not ICON_SET:
+            return
+        icon_name = node.__getitem__("target")
```

Comment:
`__getitem__` is the underlying method for the indexing operator `[]`. That is, this can just be `node["target"]`

Reply:
awesome. TIL py getters/setters